.vib/grafana/cypress/cypress/integration/grafana_test_suite.js

```diff
@@ -0,0 +1,155 @@
+/// <reference types="cypress" />
+import {
+  random,
+  verifySuccesOfAction
+} from './utils'
+
+it('allows user to log in and log out', () => {
+  cy.login();
+  cy.contains('div','Invalid username').should('not.exist');
+  cy.visit('/logout');
+  cy.get('.login-content-box').should('be.visible');
+})
+
+it('allows creating a dashboard with a panel', () => {
+  const PANEL_TITLE = 'Test panel title';
+  const PANEL_DESCRIPTION = 'Test panel description';
+  const DASHBOARD_TITLE = 'Dashboard title';
+
+  cy.login();
+  cy.visit('dashboard/new');
+  cy.contains('button','Add a new panel')
+    .should('be.visible').click();
+  cy.get('[value*="Panel Title"]').clear().type(`${PANEL_TITLE}.${random}`);
+  cy.get('textarea#description-text-area')
+    .should('exist').type(`${PANEL_DESCRIPTION}.${random}`);
+
+  cy.contains('button','Apply')
+    .should('be.visible').click();
+
+  cy.contains('.panel-title',PANEL_TITLE).should('be.visible');
+  cy.get('[aria-label*="Save dashboard"]').click();
+  cy.get('button[aria-label*="Close dialogue"]')
+    .should('be.visible');
+  cy.get('[name*="title"]').clear().type(`${DASHBOARD_TITLE}.${random}`);
+  cy.get('[type*="submit"]')
+    .should('be.visible').click();
+  verifySuccesOfAction();
+})
+
+it('checks if it is possible to upload a dashboard as JSON file', () => {
+  const DASHBOARD_TITLE = "New Test dashboard";
+  
```

Comment:
Trailing space

Reply:
Where exactly? I see nothing in line 42.